content/docs/03-cluster-profiles/5-update-profiles.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Update Packs"
+metaTitle: "Update Packs"
+metaDescription: "Learn how to update Palette packs."
+icon: " "
+hideToC: true
+fullWidth: true
+hideToCSidebar: true
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+# Overview
+
+Cluster profiles are composed of one or more pack layers. Packs can be out-of-the-box configurations, helm charts, or manifests and typically contain changes between versions, such as the addition or removal of parameters and policies. The following steps guide you in updating configurations.
+
+<br />
+
+<WarningBox>
+
+When updating to a new Kubernetes version, these rules apply:
+
+<br />
+
+- You cannot copy the pack configuration from one version to another.
+
+
+- Updating to a newer version must be done incrementally, one minor version at a time.
```

Comment:
Updating to a newer version --> Updating to a newer Kubernetes version

Reply:
Added the revision. Also removed mention of Kubernetes in the sentence that introduces the bullet list:
"When updating to a new pack version, these rules apply:"